shared/src/commonTest/kotlin/co/touchlab/kampstarter/BreedModelTest.kt

```diff
@@ -25,39 +30,46 @@ class BreedModelTest: BaseTest() {
         appStart(dbHelper, settings, ktorApi, kermit)
         dbHelper.deleteAll()
 
-        model = BreedModel(viewUpdate = { summary ->
-            if (summary.allItems.isNotEmpty())
-                itemDataSummary.complete(summary)
-        }, errorUpdate = { s ->
+        model = BreedModel(errorUpdate = { s ->
             errorString.complete(s)
         })
+
+        itemDataSummary.complete(model.selectAllBreeds().first())
     }
 
     @Test
     fun staleDataCheckTest() = runTest {
         settings.putLong(BreedModel.DB_TIMESTAMP_KEY, currentTimeMillis())
         assertFalse(ktorApi.jsonRequested)
-        model.getBreedsFromNetwork().join()
+        val deferred = async { model.getBreedsFromNetwork() }
+        deferred.await()
```

Comment:
I don't think this is doing anything different than just calling `model.getBreedsFromNetwork()` directly

Reply:
seems right, removed async